tests/test_operator.py

```diff
@@ -926,7 +926,7 @@ def test_equations_mixed_densedata_timedata(self, shape, dimensions):
         """
         grid = Grid(shape=shape, dimensions=dimensions, time_dimension=time)
         a = TimeFunction(name='a', grid=grid, time_order=2, space_order=2)
-        p_aux = Dimension(name='p_aux', size=10)
+        p_aux = Dimension(name='p_aux')
```

Comment:
Do we have a way of erroring out on extra arguments to the constructors? I suspect this was found by eyeballing the code but the presence of a `size` argument was silently ignored during execution?

Reply:
> I suspect this was found by eyeballing the code but the presence of a size argument was silently ignored during execution?

right

> Do we have a way of erroring out on extra arguments to the constructors

no, but I agree it would be useful to come up with something